graphql/src/adapters/graphql/schemas/queries.py

```diff
@@ -15,10 +20,23 @@ class Queries:
         description="""Echos the given word back as a check of basic GraphQL functionality.""")
     def get_echo(
         self,
-        word: Annotated[typing.Optional[str],  # `Optional` marks it as 'optional'...
-                        argument(
-                            description="""The word to echo back."""
-                        )
-                        ] = None,  # Default value actually MAKES it optional
+        word: Annotated[
+            typing.Optional[str],  # `Optional` marks it as 'optional'...
+            argument(
+                description="""The word to echo back."""
+            )
+        ] = None,  # Default value actually MAKES it optional
     ) -> GetEchoStrawberryResponse:
         return get_echo(word)
+
+    @field(
+        description="""Gets all phantom reports for the given filter.""")
+    def get_storage_schema_version(
+        self,
+
+    ) -> GetStorageSchemaVersionStrawberryResponse:
+        return get_storage_migration_version(
+            initialized_adapters.logger_provider.static_logger_provider.get_logger(
```

Comment:
That would cause all logged statements to look the same. The point of this structure is to differentiate logging statements from different requests.

Reply:
I am not seeing how this adds value, I would expect it to maybe be tied back to a session id or an overall execution id value for traceability. I am not sure how having a random string helps us here. Maybe a sample of what you are seeing and expecting from a log output may help me better understand here.

Reply:
Because we are using
```
class RequestIdLoggerAdapter(logging.LoggerAdapter):
    def process(self, msg: Any, kwargs: MutableMapping[str, Any]) ->\
            tuple[Any, MutableMapping[str, Any]]:
        return f"{self.extra['request_id']}: {msg}", kwargs
```
The log stack will look like:
abcd-1234-5678: Some message from user 1's request.
efgh-9876-5432: Some message from user 2's request.
abcd-1234-5678: Some error from user 1's request.

Reply:
I am open to an alternative identifier instead of the uuid, but unless it is a value that users have access to, it will have no impact.

Reply:
Here is the entirety of what we see in Cloudwatch:
2022-12-14T08:23:44.094-06:00 4f58fbfa-bd8b-451a-aade-31550d5fddf6: Running test code. Not for use in production.